dags/analyzer_helper/tests/integration/test_extract_raw_members.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime
+import unittest
+from analyzer_helper.discourse.extract_raw_members import ExtractRawMembers
+from github.neo4j_storage.neo4j_connection import Neo4jConnection
+
+class TestExtractRawMembers(unittest.TestCase):
+
+    def setUp(cls):
```

Comment:
The `setUp` class would be like `setUp(self)` instead of `setUp(cls)`. It will be called at the start of calling each of test methods